roles/sft-server/tasks/configure.yml

```diff
@@ -151,3 +151,17 @@
     owner: root
     group: root
     mode: '644'
+
+- name: create ssl certs dir
+  file:
+    path: /etc/nginx/ssl/certs
+    state: directory
+    owner: root
+    group: root
+    mode: '755'
+
+- name: download ca cert for metrics
+  get_url:
+    url: "{{ sft_metrics_ca_url }}"
+    dest: /etc/nginx/ssl/certs/metrics-ca.crt
```

Comment:
SInce this value is used [multiple times](https://github.com/wireapp/ansible-sft/pull/13/files#diff-55d4844399668cbcdfcfd8070348f5bbR30), I guess it could be defined in a variable in `vars/main.yml`.

Reply:
But then it becomes overridable, which serves no purpose.

Reply:
> But then it becomes overridable, which serves no purpose.

If you stick it in `defaults/main.yml`, yes. Not, if you put it in ` vars/main.yml`.

Reply:
So, vars are constants?

Reply:
[Yes](https://github.com/wireapp/ansible-sft/blob/develop/roles/sft-server/vars/main.yml#L1). A longer discussion around this topic started [here](https://github.com/wireapp/ansible-sft/pull/8/files#diff-7d4854bd1fef85a011d5542a9e50ec09).

Reply:
Ok, this is done.